tests/h/services/nipsa_test.py

```diff
@@ -3,73 +3,57 @@
 from h.services.nipsa import NipsaService, nipsa_factory
 
 
-@pytest.mark.usefixtures("users", "reindex_user_annotations")
+@pytest.mark.usefixtures("users")
 class TestNipsaService:
-    def test_fetch_all_flagged_userids_returns_set_of_userids(self, db_session):
-        svc = NipsaService(db_session)
-
+    def test_fetch_all_flagged_userids_returns_set_of_userids(self, svc):
         assert svc.fetch_all_flagged_userids() == {
             "acct:renata@example.com",
             "acct:cecilia@example.com",
```

Comment:
I know this wasn't your doing, but could we swap "renata", "cecilia" and "dominic" for "bad_user_1", "bad_user_2", "good_user" or something?

Much as I appreciate the colour it doesn't do anything to reveal the intent of the tests.

Reply:
:+1: